gprof2dot.py

```diff
@@ -44,6 +44,7 @@
 
 
 MULTIPLICATION_SIGN = chr(0xd7)
+TIME_FORMAT = "%.7g"
```

Comment:
Given `TIME_FORMAT` is not a constant, but rather a variable, please rename it to `timeFormat`.

Reply:
done